packages/x-components/src/x-modules/experience-controls/components/experience-controls.vue

```diff
@@ -1,36 +1,39 @@
 <script lang="ts">
-  import Vue from 'vue';
-  import { Component } from 'vue-property-decorator';
-  import { XOn } from '../../../components/decorators/bus.decorators';
+  import { defineComponent, h } from 'vue';
   import { XEvent, XEventsTypes } from '../../../wiring/events.types';
-  import { xComponentMixin } from '../../../components/x-component.mixin';
   import { experienceControlsXModule } from '../x-module';
+  import { useXBus } from '../../../composables/use-x-bus';
 
   /**
    * This component subscribes to changes in the ExperienceControls module to fire the events that
    * propagate the configuration.
    *
    * @public
    */
-  @Component({
-    mixins: [xComponentMixin(experienceControlsXModule)]
-  })
-  export default class ExperienceControls extends Vue {
-    /**.
-     * Iterates the list of XEvents received and emits them
-     *
-     * @param events - events to be emitted
-     */
-    @XOn('ExperienceControlsEventsChanged')
-    onEventsChanged(events: Partial<XEventsTypes>): void {
-      Object.entries(events).forEach(([eventName, eventPayload]) => {
-        this.$x.emit(eventName as XEvent, eventPayload);
-      });
-    }
+  export default defineComponent({
+    name: 'ExperienceControls',
+    xModule: experienceControlsXModule.name,
+    setup() {
+      const xBus = useXBus();
+
+      /**
+       * Iterates the list of XEvents received and emits them.
+       *
+       * @param events - Events to be emitted.
+       */
+      function onEventsChanged(events: Partial<XEventsTypes>): void {
+        Object.entries(events).forEach(([eventName, eventPayload]) => {
+          xBus.emit(eventName as XEvent, eventPayload);
+        });
+      }
 
-    // eslint-disable-next-line @typescript-eslint/no-empty-function
-    render(): void {}
-  }
+      xBus.on('ExperienceControlsEventsChanged', false).subscribe(event => onEventsChanged(event));
+
+      return () => {
+        return h();
+      };
```

Comment:
I think this is enough to get the same effect:

```suggestion
      return () => {};
```